src/java/org/apache/cassandra/serializers/TypeSerializer.java

```diff
@@ -55,11 +63,45 @@ public final void validate(ByteBuffer bytes) throws MarshalException
 
     public abstract Class<T> getType();
 
-    public String toCQLLiteral(ByteBuffer buffer)
+    protected String toCQLLiteralNonNull(@Nonnull ByteBuffer buffer)
```

Comment:
We could move this method down so it stays together with the other `toCQLLiteral*` methods.

Reply:
moved, didn't resolve just to make sure the placement is in-line

Reply:
Looks perfect to me, thanks